### ISSUE TITLE
Term::Form::[Win32|Linux] are basically duplicates of Term::Choose::[Win32|Linux]

### ISSUE THREAD

**Issue by mlawren** [kuerbis/Term-Form#1]
Your Term::Form:: and Term::Choose:: helper modules have mostly cosmetic differences, which seems to be an unecessary duplication. I would like to suggest folding the Form changes back into the Choose verions and using the Choose versions directly.

I'm happy to make a patch if you like.

**Comment by kuerbis** [comment]
Good suggestion, I applied it in the developer release 0.500_01.

**Comment by mlawren** [comment]
I have a couple of more suggestions to get you all the way to where Form::Linux could disappear as well:

1. Rename `Choose::Linux::__set_mode` to `__set_mode_raw`
2. Move `Form::Linux::__set_mode` to `Choose::Linux::__set_mode_cbreak`
3. Make `Term::Form` use `Choose::Linux` as the Linux plugin and convert all `__set_mode` calls to `__set_mode_cbreak`, and change all `__reset_mode` calls to use the 3-argument version.
4. Remove `Form::Linux` completely 

More generally, it is quite hard to follow your commits. It would be much clearer if you perhaps:

1. Separated code modifications from version changes
2. The natural outcome of step 1 might be that your commit messages say what the commit does :-)

**Comment by mlawren** [comment]
Thanks for the quick response, and for publishing the code in general!
